common/lib/xmodule/xmodule/js/src/html/edit.js

```diff
@@ -124,10 +124,16 @@
           Disable visual aid on borderless table.
            */
           visual: false,
-          plugins: "textcolor, link, image, codemirror",
+          plugins: "textcolor, link, image, codemirror, s3url",
           codemirror: {
             path: baseUrl + "js/vendor"
           },
+          s3url: {
+            linktypes: process.env.S3URL_PLUGIN_LINKTYPES.split(','),
```

Comment:
Following on from the previous comment, then this could become `linktypes: process.env.NODE_ENV_VARIABLES.S3URL_PLUGIN_LINKTYPES.split(','),` and so on...

Reply:
:+1: